csrc/anemone_mempool.h

```diff
@@ -34,15 +38,30 @@ typedef struct {
   void     *maximum_ptr;
 } anemone_mempool_t;
 
+
+ANEMONE_INLINE
+size_t
+round_up_allocation (size_t num_bytes)
```

Comment:
Any reason the return type `size_t` is on its own line? I don't have a problem with this style per se, but it's inconsistent with the rest of the file / project.